timeseries-streaming/timeseries-java-applications/Examples/build.gradle

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id 'java'
+    id 'idea'
+    id "com.diffplug.gradle.spotless" version "3.24.2"
+}
+
+sourceCompatibility = 1.8
+
+dependencies {
+
+    compile project(':TimeSeriesPipeline')
+    compile project(':TimeSeriesMetricsLibrary')
+
+    compile group: 'org.apache.beam', name: 'beam-sdks-java-core', version: "${beamVersion}"
+    compile group: 'org.apache.beam', name: 'beam-sdks-java-io-google-cloud-platform', version:"${beamVersion}"
+    compile group: 'org.apache.beam', name: 'beam-runners-direct-java', version: "${beamVersion}"
+    compile group: 'org.apache.beam', name: 'beam-runners-google-cloud-dataflow-java', version:  "${beamVersion}"
+
+    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-csv', version: '2.11.1'
+    compile group: 'org.slf4j', name: 'slf4j-jdk14', version: '1.7.7'
+
+    compile             "com.google.auto.value:auto-value-annotations:${autoValueVersion}"
+    annotationProcessor "com.google.auto.value:auto-value:${autoValueVersion}"
+
+    // Enable code formatting
+    compile 'com.google.googlejavaformat:google-java-format:1.7'
+
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
+    testCompile group: 'org.apache.beam', name: 'beam-runners-direct-java', version: "${beamVersion}"
+}
+
+// Enable code formatting
+spotless {
+    java {
+        licenseHeaderFile '../Apache_Licence_For_Spotless'
+        googleJavaFormat('1.7')
+    }
+}
+
+task forex_example(type: JavaExec) {
```

Comment:
I dont think we have done this yet, but maybe we should look at adding notes on any tasks that we add. What they do etc...